sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet/ssd_entry_point.py

```diff
@@ -260,44 +179,55 @@ def model_fn(model_dir):
     :param: model_dir The directory where model files are stored.
     :return: a model (in this case a Gluon network)
     """
+    import neomxnet  # noqa: F401
     net = gluon.SymbolBlock.imports(
-        '%s/model-symbol.json' % model_dir,
+        '%s/compiled-symbol.json' % model_dir,
         ['data'],
-        '%s/model-0000.params' % model_dir,
+        '%s/compiled-0000.params' % model_dir,
     )
+    #run warm-up inference on empty data
+    ctx = mx.gpu() if mx.context.num_gpus() > 0 else mx.cpu()
```

Comment:
compiled models only work with whatever they were compiled for, so just pick GPU (since were compiling for P3 in this example). This code implies that if GPU is not available it will fall back to CPU (which isnt true, it will fail)

Reply:
done